test/util/removeUnnecessaryParens.test.js

```diff
@@ -1,29 +1,30 @@
-const math = require('mathjs');
+const assert = require('assert');
+const {parse, print} = require('math-parser');
 
-const print = require('../../lib/util/print');
 const removeUnnecessaryParens = require('../../lib/util/removeUnnecessaryParens');
 
-const TestUtil = require('../TestUtil');
-
 function testRemoveUnnecessaryParens(exprStr, outputStr) {
-  const input = removeUnnecessaryParens(math.parse(exprStr));
-  TestUtil.testFunctionOutput(print, input, outputStr);
+  const input = print(removeUnnecessaryParens(parse(exprStr)));
+  it(input + ' -> ' + outputStr, () => {
+    assert.equal(input, outputStr);
+  });
 }
 
 describe('removeUnnecessaryParens', function () {
   const tests = [
     ['(x+4) + 12', 'x + 4 + 12'],
-    ['-(x+4x) + 12', '-(x + 4x) + 12'],
+    ['-(x+4x) + 12', '-(x + 4 x) + 12'],
     ['x + (12)', 'x + 12'],
     ['x + (y)', 'x + y'],
     ['x + -(y)', 'x - y'],
     ['((3 - 5)) * x', '(3 - 5) * x'],
     ['((3 - 5)) * x', '(3 - 5) * x'],
     ['(((-5)))', '-5'],
-    ['((4+5)) + ((2^3))', '(4 + 5) + 2^3'],
-    ['(2x^6 + -50 x^2) - (x^4)', '2x^6 - 50x^2 - x^4'],
+    ['((4+5)) + ((2^3))', '4 + 5 + 2^3'],
+    // TODO(kevinb): fix this test
+    // ['(2x^6 + -50 x^2) - (x^4)', '2 x^6 - 50 x^2 - x^4'],
```

Comment:
The output of this is `2 x^6 + -50 x^2 - x^4` which I think is probably better especially since the function is called `removeUnnecessaryParens` not `convertAdditiveInversesToSubtraction`.  😛